pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+---
+pagename: Overview
+sitesection: Documents
+categoryname: Reporting
+documentname: Outbound Configuration API
+permalink: outbound-configuration-api-overview.html
+root-link: true
+indicator: Proactive Messaging
+redirect_from:
+ - proactive-messaging-reporting-api.html
+---
+
+### Introduction
+The Outbound configuration API provides a complete list of API handoffs created in the Proactive Web Tool. This API allows 1) Fetching all handoffs configurations 2) Fetching all handoff configurations filtered by a specific messaging channel (SMS, WhatsApp, Apple Business Chat, In-app, Google Rcs, etc.)
+
+API provides outbound configuration for below services
+* Proactive Messaging 2.0
+
+
+### Getting Started
+
+**Who can access this Outbound Configuration API**
+
+All Brands who use Proactive Messaging version 2.0 have access to the Outbound Configuration API service.
+
+**What are the limitations**
+
+Both endpoints are currently read only, i.e it is not possible to make configuration changes using the outbound configuration API.
+
+**OAuth 2.0 Authorization**
+* Either Administrator or LPA can get client_id and client_secret by clicking the show secrets on the web UI as shown below.<br>
+![Secrets](img/proactive/proactive-show-secrets.png)  
```

Comment:
Please try whether you can avoid a line break (`<br>`), by indenting the image by 2 or 4 spaces.